pkg/services/ngalert/notifier/multiorg_alertmanager.go

```diff
@@ -29,12 +30,43 @@ var (
 	ErrAlertmanagerNotReady = fmt.Errorf("Alertmanager is not ready yet")
 )
 
+type Alertmanager interface {
```

Comment:
Now realizing this is _both implemented and consumed_ by the multi-org alertmanager. :thinking:

Reply:
Yep, it's defined, implemented, and consumed here. All methods are used in the `notifier` package, and this interface (minutes the methods labeled "State") was defined in the `api` package.

The interface will be implemented by the proxy Alertmanager in the next PR. I agree that the interface is huge, I'll try to address that problem when adding the new implementation.

Reply:
I think the big interface in the notify package in this case is fine because it just facilitates polymorphism. However, I think we may keep the interface in api package. This will allow us to have more focused contract between notify and api packages.

Reply:
@yuri-tceretian the `api` package imports the `notifier` package. Since we're using the `Alertmanager` interface in the multi-org Alertmanager, leaving the interface in `api` results in an import cycle.

Reply:
Oh, that's unfortunate. Ok, let's leave it this way for now.